src/main/java/it/gov/pagopa/pu/pagopapayments/exception/PagopaPaymentsExceptionHandler.java

```diff
@@ -59,6 +59,11 @@ public ResponseEntity<PagoPaPaymentsErrorDTO> handleRuntimeException(RuntimeExce
     return handleException(ex, request, HttpStatus.INTERNAL_SERVER_ERROR, PagoPaPaymentsErrorDTO.CodeEnum.PAGOPA_PAYMENTS_GENERIC_ERROR);
   }
 
+  @ExceptionHandler({NotPayableSilActualizedAmountException.class})
+  public ResponseEntity<PagoPaPaymentsErrorDTO> handleNotPayableSilActualizedAmountException(RuntimeException ex, HttpServletRequest request) {
+    return handleException(ex, request, HttpStatus.CONFLICT, PagoPaPaymentsErrorDTO.CodeEnum.PAGOPA_PAYMENTS_GENERIC_ERROR);
```

Comment:
we will catch this exception, so it would never be handled here...
we could think to let it be here, but it's not correct `PAGOPA_PAYMENTS_GENERIC_ERROR`, we should introduce a specific code, eg NOT_PAYABLE